app/components/chart-published.tsx

```diff
@@ -142,13 +159,22 @@ export const ChartPublished = (props: ChartPublishedProps) => {
                 <Description text={state.layout.meta.description[locale]} />
               )}
             </Box>
-
-            <ChartPanelLayout
-              layoutType={state.layout.layout}
-              chartConfigs={state.chartConfigs}
-              renderChart={renderChart}
-            />
-          </>
+            <Flex
+              style={{
+                gap: 16,
+              }}
+              sx={{
+                flexDirection: "column",
+              }}
```

Comment:
I think we shouldn't mix up `style` and `sx` props 👀 You could change it to `<Flex flexDirection="column" gap={4}>`

also see [theme spacings](https://github.com/visualize-admin/visualization-tool/blob/03f3eef58b288fe25dc8784b55dee54763550c1c/app/themes/federal.tsx#L110) for shortcut values, based on index 👍

Reply:
> I think we shouldn't mix up `style` and `sx` props 👀 You could change it to `<Flex flexDirection="column" gap={4}>`
> 
> 
> 
> also see [theme spacings](https://github.com/visualize-admin/visualization-tool/blob/03f3eef58b288fe25dc8784b55dee54763550c1c/app/themes/federal.tsx#L110) for shortcut values, based on index 👍 

Thanks, I'm not too familiar with mui, have been using other libs in the past.

Reply:
Sure, np, it's not a huge thing, but it's helpful to know some of the benefits we get through it 👍